configs/projects/bolt-runtime.rb

```diff
@@ -107,7 +107,7 @@
   # --------------
 
   # Ruby and deps
-  proj.component "openssl-#{proj.openssl_version}"
+  proj.component "openssl-#{proj.openssl_version}" unless platform.name =~ /^redhatfips-7-.*/  # BOLT-1339 FIPS uses system SSL libs
```

Comment:
I think you'll need to do a bit more than this to make sure curl, etc. build correctly. Try grepping this repo for `:system_openssl` -- the agent runtimes use that setting to do this filtering instead of a specific platform name.

Reply:
That setting just controls a conditional at a lower level that includes the component or not.

Reply:
You will still need to update the ca bundle component to use the system openssl. It already look for the `:system_openssl` setting -- did you try the grep?

Reply:
In progress.